src/ducks/transactions/actions/ReimbursementStatusAction/ReimbursementStatusModal.jsx

```diff
@@ -13,91 +16,74 @@ import { isHealthExpense } from 'ducks/categories/helpers'
 import flag from 'cozy-flags'
 import ContactItem from 'ducks/transactions/actions/ReimbursementStatusAction/ContactItem'
 
-class _ReimbursementStatusModal extends React.PureComponent {
-  render() {
-    const {
-      transaction,
-      onChange,
-      t,
-      className,
-      breakpoints: { isMobile },
-      brands,
-      ...rest
-    } = this.props
-    const choices = ['pending', 'reimbursed', 'no-reimbursement']
-    const status = getReimbursementStatus(transaction)
+const ReimbursementStatusModal = props => {
```

Comment:
to keep the same behavior you should have been using React.memo, no?

Reply:
Done